Src/BootCamp.Chapter/Computer/Motherboard.cs

```diff
@@ -2,5 +2,8 @@
 {
     public class Motherboard
     {
+        public Motherboard()
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this redundant constructor.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5121609)